README.md

```diff
@@ -1,137 +1,204 @@
 
-[asgi]: https://asgi.readthedocs.io/
-[starlette]: https://www.starlette.io
 [tartiflette]: https://tartiflette.io
+[starlette]: https://www.starlette.io
 
-> ⚠️ This package is still under development. Be sure to pin your dependencies!
-
-## Installation
-
-Assuming [you have already installed Tartiflette](https://tartiflette.io/docs/tutorial/install-tartiflette), you can install `tartiflette-starlette` from PyPI:
-
-```bash
-pip install tartiflette-starlette
-```
+The included `TartifletteApp` is an ASGI wrapper with the following features:
 
-**Note**: `tartiflette-starlette` is compatible with Starlette 0.12+.
+- Compatible with any ASGI server and framework.
+- Standalone and sub-app serving.
+- Built-in [GraphiQL] client.
 
-## Usage
+[graphiql]: https://github.com/graphql/graphiql
 
-> Getting started with Tartiflette or GraphQL? See [Resources](#resources).
+**Note**: WebSocket subscriptions aren't supported yet.
 
-The `TartifletteApp` class provided by `tartiflette-starlette` is an ASGI3-compliant application. As such, it can be served on its own using any ASGI web server, or it can be mounted onto another ASGI application.
+**Table of contents**
 
-**Note**: GraphQL subscriptions are not supported yet.
+- [Quickstart](#quickstart)
+- [Installation](#installation)
+- [User guide](#user-guide)
+- [API Reference](#api-reference)
+- [FAQ](#faq)
 
-### Create an ASGI GraphQL app
+## Quickstart
 
 ```python
-# graphql.py
 from tartiflette import Resolver
 from tartiflette_starlette import TartifletteApp
 
 @Resolver("Query.hello")
-async def resolve_hello(parent, args, context, info):
+async def hello(parent, args, context, info):
     name = args["name"]
     return f"Hello, {name}!"
 
 sdl = """
-    type Query {
-        hello(name: String): String
-    }
+  type Query {
+    hello(name: String): String
+  }
 """
 
 app = TartifletteApp(sdl=sdl)
 ```
 
-> **Note**: the GraphQL endpoint is exposed on `/` by default.
-
-### Serve the app
-
-A `TartifletteApp` can be served using any ASGI web server, e.g. [uvicorn]:
+Save the file as `graphql.py` and start a [uvicorn] server:
 
 [uvicorn]: https://www.uvicorn.org
 
 ```bash
 uvicorn graphql:app
 ```
 
-### Make GraphQL queries
-
-As per the [GraphQL spec](https://graphql.org/learn/serving-over-http/#http-methods-headers-and-body), the GraphQL query can be passed in various ways:
+> **Note**: the GraphQL endpoint is exposed on `/` by default.
 
-- **URL query string** (methods: `GET`, `POST`):
+Make a request:
 
 ```bash
-curl 'http://localhost:8000?query=\{hello(name:"Chuck")\}'
+curl -H "Content-Type: application/graphql"  -d '{ hello(name: "Chuck") }' http://localhost:8000
 ```
 
-- **JSON-encoded body** (methods: `POST`):
+Response:
 
-```bash
-curl \
-  -X POST \
-  -H "Content-Type: application/json" \
-  -d '{"query": "{ hello(name: \"Chuck\") }"}' \
-  http://localhost:8000
+```json
+{ "data": { "hello": "Hello, Chuck!" }, "errors": null }
```

Comment:
I think there is a little mistake here. In a GraphQL response, if no errors were encountered during the requested operation, the errors entry should not be present in the result (cf. [spec](https://graphql.github.io/graphql-spec/June2018/#sec-Errors)).
```suggestion
{ "data": { "hello": "Hello, Chuck!" } }
```
The GraphiQL image below should be updated to 🙏

Reply:
Good catch! I’ll change these. :-)

Reply:
Tracked in #28, if that's fine with you I suggest we leave this as is and I'll update it when #28 is solved (which will all happen before releasing 0.2).

Reply:
Just been fixed, actually. :+1: